src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentOrgPlan.spec.tsx

```diff
@@ -70,6 +126,131 @@ describe('CurrentOrgPlan', () => {
     })
   })
 
+  describe('when plan update success banner should be shown', () => {
+    // beforeEach(() => {
```

Comment:
Same here

Reply:
oops, sorry forgot to cleanup. Yeah I'll delete all this crap